src/city_score/sources/traillink.py

```diff
@@ -0,0 +1,87 @@
+from ..cache import cache
+from ..decorators import dimension, scorer
+from ..source import Source
+from html.parser import HTMLParser
+import requests
+
+activities_filter = {
+    'Atv': 'ATV',
+    'Bike': 'BIKE',
+    'Birding': 'BIRD',
+    'Cross Country Skiing': 'XSKI',
+    'Dog Walking': 'DOG',
+    'Fishing': 'FISH',
+    'Geocaching': 'GEO',
+    'Hiking': 'HIKE',
+    'Horseback Riding': 'HORSE',
+    'Inline Skating': 'SKTS',
+    'Mountain Biking': 'MTBK',
+    'Running': 'RUN',
+    'Snowmobiling': 'SNOW',
+    'Walking': 'WALK',
+    'Wheelchair Accessible': 'WHEEL'
+}
```

Comment:
@jacobbudin added this activities filter that could be useful, lmk your thoughts
<img width="332" alt="image" src="https://github.com/jacobbudin/city-score/assets/993698/99a94f41-5220-4bce-8b16-34a0c0751076">

Reply:
Good addition. 👍🏻